.github/workflows/smoke.yml

```
@@ -0,0 +1,61 @@
+name: Smoke Test
+   
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches-ignore:
+      - 'tests-passed'
+    
+jobs:
+  smoke_test:
+    name: "smoke test"
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+
+      - name: Install unbuffer
+        run: sudo apt-get install -yqq expect
+
+      - name: Boot dev image
+        run: unbuffer d/boot_dev
+
+      - name: tmp
+        run: sudo chown -R 1000:1000 .
+
+      - name: Bundle
+        run: unbuffer d/bundle install
+
+      - name: Migrate
+        run: unbuffer d/rake db:migrate
+
+      - name: Create script
+        run: | 
+          touch tmp/script.rb
+          echo "User.create!(username:'smoke_user', password:'P4ssw0rd', admin:true)" >> tmp/script.rb
```

Why not directly call the rails console?

```suggestion
          echo "User.create!(username:'smoke_user', password:'P4ssw0rd', admin:true)" | d/rails c
```

---

The `d/rails` wrapper doesn't hook up stdin 😢